src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -228,7 +228,16 @@ public void TestStartOnUnixWithBadPermissions()
         {
             string path = GetTestFilePath();
             File.Create(path).Dispose();
-            Assert.Equal(0, chmod(path, 644)); // no execute permissions
+            int mode;
+            if (PlatformDetection.IsFreeBSD)
+            {
+                mode = Convert.ToInt32("644", 8);
```

Comment:
Looks like a bug, on UNIX we use octals for `mode`.

Reply:
I can test that @janvorli and fix it if you are right for Linux and OSX. This one really got me surprised as the call was failing with Invalid arguments errno

Reply:
It did not fail but the permission is not right:
    int ret = chmod("/tmp/aa", 777) ;

macik:tmp furt$ ls -al aa
-r----x--t  1 furt  wheel  0 Oct 12 17:04 aa

I guess that was never found since both tests expect exec() to fail.